query-engine/query-engine/src/request_handlers/graphql/schema_renderer/field_renderer.rs

```diff
@@ -45,7 +45,8 @@ impl GqlFieldRenderer {
         };
 
         let rendered_type = field.field_type.into_renderer().render(ctx);
-        format!("{}{}: {}", field.name, rendered_args, rendered_type)
+        let bang = if !field.is_nullable { "!" } else { "" };
+        format!("{}{}: {}{}", field.name, rendered_args, rendered_type, bang)
```

Comment:
Note: This will render nullable lists as `[String]` and non-nullable lists as `[String]!`. Even though we technically can't have nullable lists AFAIK.

Reply:
If it's only GQL we don't care, it's merely a debug tool.